### PR TITLE
Update Windows documentation

### PR DESCRIPTION
**Pull request by alinbalutoiu**
This patch updates the Windows documentation for
the OVN/OVS CNI plugin to reflect the latest changes.

It also updates the documentation with details on
how to create services from kubelet and kube-proxy on
Windows.

Signed-off-by: Alin Balutoiu <abalutoiu@cloudbasesolutions.com>

### CODE REVIEW

**Comment by alinbalutoiu** [comment]
@aserdean @bsteciuk @michmike please take a look.

File: content/en/docs/getting-started-guides/windows/_index.md [hunk 1]
```diff
@@ -176,49 +182,115 @@ For an unattended installation please use the following command:
...
+ovs-vsctl --version
+```
 
 For overlay, OVS on Windows requires a transparent docker network to function properly. Please use the following to create a transparent docker network which will be used by OVS. From powershell:
 ```
 docker network create -d transparent --gateway $GATEWAY_IP --subnet $SUBNET `
     -o com.docker.network.windowsshim.interface="$INTERFACE_ALIAS" external
 ```
-Where $SUBNET is the minion subnet which will be used to spawn pods on (the one which will be used by kubernetes), $GATEWAY_IP is the first IP of the $SUBNET and $INTERFACE_ALIAS is the interface used for creating the overlay tunnels (must have connectivity with the rests of the OVN hosts).
+Where $INTERFACE_ALIAS is the interface used for creating the overlay tunnels (must have connectivity with the rests of the OVN hosts). The variables $SUBNET and $GATEWAY_IP, and how to fetch their values is explained below.
```
Comment (aserdean) [inline content/en/docs/getting-started-guides/windows/_index.md]: s/Where $INTERFACE_ALIAS/$INTERFACE_ALIAS

Also I think you should add a new line before: "The variables..."

File: content/en/docs/getting-started-guides/windows/_index.md [hunk 2]
```diff
@@ -176,49 +182,115 @@ For an unattended installation please use the following command:
...
+
+The subnets for each minion node will be assigned automatically by OVN. This requires the Windows node to get registered in Kubernetes in order to be assigned a POD subnet. Please start the kubelet and wait until the node gets registered.
```
Comment (aserdean) [inline content/en/docs/getting-started-guides/windows/_index.md]: Maybe rephrase it a bit. I.e.:

The subnets for each minion node will be assigned automatically by OVN once the node registers in `Kubernetes`. To achieve the latter simply start the `kubelet` and wait until it registers.

File: content/en/docs/getting-started-guides/windows/_index.md [hunk 3]
```diff
@@ -489,6 +489,40 @@ spec:
         - containerPort: 80
 ```
 
+### Kubelet and kube-proxy can now run as Windows services
+
+Starting with kubernetes v1.11, kubelet and kube-proxy can run as Windows services.
+
+This means that you can now register them as Windows services via `sc` command. More details about how to create Windows services with `sc` can be found [here](https://support.microsoft.com/en-us/help/251192/how-to-create-a-windows-service-by-using-sc-exe).
+
+**Examples:**
```
[the remark this answers is not in the document]
Comment (alinbalutoiu) [reply content/en/docs/getting-started-guides/windows/_index.md]: Just curious, why should this be a level 4 heading? All the other features have level 3 heading.

[the remark this answers is not in the document]
Comment (zacharysarah) [reply content/en/docs/getting-started-guides/windows/_index.md]: @alinbalutoiu Are the examples under `### Kubelet and kube-proxy can now run as Windows services`? If so, it's clearer to format the heading as `#### Examples` rather than just bolding the text.

[the remark this answers is not in the document]
Comment (alinbalutoiu) [reply content/en/docs/getting-started-guides/windows/_index.md]: @zacharysarah Yes the examples are under ``### Kubelet and kube-proxy can no run as Windows services``. I wanted to stick with the previous format (example: https://github.com/kubernetes/website/pull/8454/files#diff-10523ec8ea4ab689a6b2b75bf6e3a5c5R363). If you think that way is clearer I can update afterwards to replace with ``#### Examples`` both usages.
